src/renderer/viz/expressions/basic/variable.js

```diff
@@ -13,13 +13,13 @@ import { checkString } from '../utils';
  *   variables: {
  *     sum_price: s.clusterSum(s.prop('price'))
  *   }
- *  filter: s.neq(s.var('sum_price'), 'london'),
+ *  filter: s.neq(s.var('sum_price'), 'london')
  * });
  *
  * @example <caption>(String)</caption>
  * const viz = new carto.Viz(`
- *   @sum_price: clusterSum($price)
- *   filter: @sum_price !== 'london'
+ *   \@sum_price: clusterSum($price)
```

Comment:
Why do we need the `\`?

Reply:
Because without \, @ is not parsed by JSDoc
![selection_256](https://user-images.githubusercontent.com/2227572/44261000-8807e880-a216-11e8-8ab7-ed8da4f6bf29.png)